react-components/src/components/AccountContainer/accountContainer.js

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ *
+ *    Copyright 2019 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { useTranslation, Trans } from 'react-i18next';
+
+import { useUserContext } from '../../context/UserContext';
+
+import AccountTrigger from './accountTrigger';
+import AccountDropdown from './accountDropdown';
+
+const container = document.querySelector('.header__accountTrigger #miniaccount');
```

Comment:
For re-useability, can the selector be a prop on the component and the query is done inside the component?

Reply:
I think we'd better pass all the selectors in a config object for the whole app

Reply:
Should live in https://github.com/adobe/aem-cif-guides-venia/pull/12 then, so it can be easier customized by customers :)

Reply:
So do we want to deal with this later by making changes in https://github.com/adobe/aem-cif-guides-venia? @herzog31 @dplaton ?

Reply:
Hi @herzog31, the query selector is now passed via `<CommerceApp>`, please review :)